ewcosmeticitem.py

```diff
@@ -173,3 +174,34 @@ def __init__(
 		await ewutils.send_message(cmd.client, cmd.message.channel, ewutils.formatMessage(cmd.message.author, response))
 	else:
 		await ewutils.send_message(cmd.client, cmd.message.channel, ewutils.formatMessage(cmd.message.author, 'You need to specify which cosmetic you want to paint and which dye you want to use! Check your **!inventory**.'))
+
+async def smoke(cmd):
```

Comment:
potential problems: 1) when you smoke a cigarette you then get a cigarette butt. the next time you smoke the search will return that cigarette butt and you won't be able to smoke even if you still have unused cigarettes
2) if the bot restarts while you're smoking it will never go out. might be funny actually

Reply:
took care of #1, but let's leave #2 in. olympic torch cigs sound like fun.